src/config/topics.js

```diff
@@ -37,4 +37,41 @@ export const casa = {
   projection: 'EPSG:3857',
 };
 
-export default [netzkarte];
+export const bauprojekte = {
+  name: 'ch.sbb.construction',
+  key: 'ch.sbb.construction',
+  linkUrl: 'https://maps.trafimage.ch/#/ch.sbb.construction',
```

Comment:
Permalink should take care of changing the url when the topic change. This link  doesn't seem a robust solution.